sagemaker_batch_transform/custom_tensorflow_inference_script_csv_and_tfrecord/custom_tensorflow_inference_script_csv_and_tfrecord.ipynb

```diff
@@ -0,0 +1,717 @@
+{
```

Comment:
<p>"I.e. the CSV data and TFRecord identical they are just of different formats" should be "The CSV and TFRecord use identical data, they just express it in different formats."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3438/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
Done